Create FileArchive schema and ContentTypes for the EBRAINS image service

lzehl opened the pull request:
We introduce the FileArchive schema to point to specifically structured file location containing typically a large number of files, that should not be indexed by a graph database.

Use case within EBRAINS: image service buckets / containers for Neuroglancer or DCI Capture (@xgui3783 correct?) 
The respective content types are attached to the PR (@apdavison & @jagru20 please help with the review)

@marcnm @alexisdurieux @dickscheid FYI 

[the background of this PR is also discussed in #318]

lzehl commented:
@olinux we need to have some clarification for the image-service bucket with the DZI files. the rest is ready to be reviewed.

jagru20 commented on instances/data/contentTypes/vnd.ebrains.image-service.microsoft-deep-zoom.jsonld:
```diff
@@ -0,0 +1,11 @@
+	},
+	"@id": "https://openminds.ebrains.eu/instances/contentTypes/application/vnd.ebrains.image-service.microsoft-deep-zoom",
+	"@type": "https://openminds.ebrains.eu/core/ContentType",
+  "description": "This content type defines a file repository produced by the EBRAINS image-service holding image files that are directly ingestable by Microsoft Deep Zoom.",
```
indentation

lzehl replied:
corrected

jagru20 commented on instances/data/contentTypes/vnd.ebrains.image-service.neuroglancer.jsonld:
```diff
@@ -0,0 +1,11 @@
+	},
+	"@id": "https://openminds.ebrains.eu/instances/contentTypes/application/vnd.ebrains.image-service.neuroglancer",
+	"@type": "https://openminds.ebrains.eu/core/ContentType",
+  "description": "This content type defines a file repository produced by the EBRAINS image-service holding image files that are directly ingestable by Neuroglancer.",
```
indentation

lzehl replied:
done

jagru20 commented on instances/data/contentTypes/vnd.ebrains.image-service.microsoft-deep-zoom.jsonld:
```diff
@@ -0,0 +1,11 @@
+	"@type": "https://openminds.ebrains.eu/core/ContentType",
+  "description": "This content type defines a file repository produced by the EBRAINS image-service holding image files that are directly ingestable by Microsoft Deep Zoom.",
+	"name": "application/vnd.ebrains.image-service.microsoft-deep-zoom",
+	"relatedMediaType": null,
+	"synonym": null
```
I think if they are null, they can be omitted as they are not required.

lzehl replied:
this is true, I usually leave them in though for the controlled terms, content types and licenses because we should fill them at one point (as a reminder for us)

lzehl replied:
I've deleted "relatedMediaType" (which we will not have for this case) and added a synonym

jagru20 commented on instances/data/contentTypes/vnd.ebrains.image-service.neuroglancer.jsonld:
```diff
@@ -0,0 +1,11 @@
+	"@type": "https://openminds.ebrains.eu/core/ContentType",
+  "description": "This content type defines a file repository produced by the EBRAINS image-service holding image files that are directly ingestable by Neuroglancer.",
+	"name": "application/vnd.ebrains.image-service.neuroglancer",
+	"relatedMediaType": null,
+	"synonym": null
```
I think if they are null, they can be omitted as they are not required.

lzehl replied:
this is true, I usually leave them in though for the controlled terms, content types and licenses because we should fill them at one point (as a reminder for us)

lzehl replied:
I've deleted "relatedMediaType" (which we will not have for this case) and added a synonym

jagru20 reviewed (changes requested):
I am a bit limited in knowledge about these repository types, thus I am not able to evaluate if the names are chosen appropriately but from the description provided it seems plausible. 
Found some minor cosmetic issues.

lzehl commented:
@jagru20 I'm not yet happy about the content types. I will most likely change them if I get the feedback I need.

lzehl commented:
@olinux @jagru20 @apdavison I've added the content types of Microsoft Deep Zoom software also to this PR since they were missing in our list.

lzehl commented:
@jagru20 @olinux @apdavison: 
**The PR is ready to be reviewed again.** With the help of @xgui3783 I've updated the content types for the image service and, for completion and references, added the individual content types of the related software (deep zoom and neuroglancer).

@marcnm @alexisdurieux In the long run would be good if you could online document the specifications for each of the image-service bucket/container types so that we can add a specification link to the respective content types.
